Documentation/building/windows-instructions.md

```diff
@@ -19,8 +19,13 @@ We also require that [Visual Studio 2015 Update 1](https://www.visualstudio.com/
 ## Building From the Command Line
 
 Open a [Visual Studio Command Prompt](http://msdn.microsoft.com/en-us/library/ms229859(v=vs.110).aspx).
-From the root of the repository, type `build.cmd`. This will build everything and run
-the core tests for the project. 
+
+`build.cmd` - Will cause basic tool initialization and build the default configuration for refs, libs, and packages.
```

Comment:
It is about build time and about separating concerns. I think building the product should be independent from testing the product. Building the tests is actually one form of testing the product, in particular the API surface area, and then running the tests is testing the behavior of the product. I've never really liked that we made building and running the inner loop tests part of the default build.cmd so I wanted to take this opportunity try and split them with this work, and it does directly add value for building just the product from source for the Build from Source effort. 

With that said if we discover it causes issues we can consider changing the default and I'm also fine if we want to add an option to build.cmd to have it also build and run the tests. As I do agree it is nice to have one script to do all my work in one command. I filed https://github.com/dotnet/corefx/issues/15116 to track that.